code/controllers/subsystem/discord.dm

```diff
@@ -36,15 +39,22 @@ SUBSYSTEM_DEF(discord)
 	var/list/notify_members_cache = list()
 	/// People to notify on roundstart
 	var/list/people_to_notify = list()
-	/// List that holds accounts to link, used in conjunction with TGS
-	var/list/account_link_cache = list()
-	/// list of people who tried to reverify, so they can only do it once per round as a shitty slowdown
-	var/list/reverify_cache = list()
+
+	/// People who have tried to verify this round already
+	var/list/reverify_cache
+
+	/// Common words list, used to generate one time tokens
+	var/static/list/common_words
```

Comment:
just realised this probably doesn't work, since it's static? and initiated later?

Reply:
Why static anyway if subsystems are only initialized once?

Reply:
I dunno, paranoia, faint hope it's more effective?

Reply:
Just make it not-static and it should work as intended.